lib/maestro/aggregate/root.ex

```diff
@@ -322,9 +343,10 @@ defmodule Maestro.Aggregate.Root do
   def eval_command(agg, command) do
     with seq <- Map.get(command, :sequence, 0),
          agg <- update_aggregate(agg, seq),
-         module <- lookup_module(agg.command_prefix, command.type),
-         events <- module.eval(agg, command) do
-      persist_events(agg, command, events)
+         com_module <- lookup_module(agg.command_prefix, command.type),
+         events <- com_module.eval(agg, command),
+         projections <- prepare_strong_projections(agg, events) do
```

Comment:
Are we comfortable crashing in this pipeline? Since we're delegating to user defined modules here its entirely possible that even with stateless code they could get bad pattern matches or other exceptions. If that happens then we crash. Which may be totally fine. I'm not familiar enough with this code to know.